src/middleware/authorize.middleware.ts

```diff
@@ -26,7 +35,13 @@ const authorizeMiddleware = (
   }
 
   try {
-    const _decoded = jwt.verify(token, process.env.JWT_SECRET!);
+    const decoded = jwt.verify(token, process.env.JWT_SECRET!) as AuthDecoded;
+
+    req.user = {
+      _id: new mongoose.Types.ObjectId(decoded._id),
```

Comment:
because the content in token is " id " not " _id ",so you should change to decoded.id, and also change AuthDecoded declare form _id to id